packages/transaction-controller/src/TransactionController.ts

```diff
@@ -467,6 +470,22 @@ export type TransactionControllerTransactionNewSwapAndSendEvent = {
   payload: [{ transactionMeta: TransactionMeta }];
 };
 
+/**
+ * Represents the `TransactionController:transactionNewBridgeApproval` event.
+ */
+export type TransactionControllerTransactionNewBridgeApprovalEvent = {
```

Comment:
In an ideal world, the controller wouldn't be coupled to specific transaction types in the events since we already have the `unapprovedTransactionAdded`.

Could we instead just subscribe to that in the client and check the `type` property?

Reply:
Yea we can, I was just referencing the similar Swap types here initially.